cryodrgn/models.py

```diff
@@ -232,6 +232,12 @@ def eval_volume(self, *args, **kwargs):
         assert isinstance(module, Decoder)
         return module.eval_volume(*args, **kwargs)
 
+    def forward(self, *args, **kwargs):
+        return self.dp.module.forward(*args, **kwargs)
+
+    def state_dict(self, *args, **kwargs):
```

Comment:
This fix allows the state_dict of the decoder in `train_nn` to be saved correctly in the `--multigpu` case.